script.js

```diff
@@ -0,0 +1,89 @@
+//shortcut
+
+function byId(id) {
+    return document.getElementById(id);
+}
+
+function set(key, data) {
```

Comment:
What does `set` mean? What is being set? I should know by just looking at your function name.

Reply:
okay sir